src/makieplotting.jl

```diff
@@ -566,6 +566,156 @@ function ferriteviewer(plotter::MakiePlotter, data::Vector{Vector{T}}) where T
     display(fig)
 end
 
+#function get_domain(ip::Ferrite.Interpolation{ref_shape,order}, meshsize::Int) where {ref_shape<:Union{RefQuadrilateral,RefTriangle},order}
+function get_domain(ip::Ferrite.Interpolation{2,ref_shape,order}, meshsize::Int) where {ref_shape<:Union{Ferrite.RefCube,Ferrite.RefTetrahedron},order}
+    rcs = Ferrite.reference_coordinates(ip)
+    lwr = min(vcat(collect.(rcs)...)...)
+    upr = max(vcat(collect.(rcs)...)...)
+    y = range(upr,lwr,length=meshsize)
+    x = range(lwr,upr,length=meshsize)
+    ref_z = get_ref_z(ip,meshsize)
+    return x,y,ref_z
+end
```

Comment:
Why not have 2 routines, one for a reference triangle and one for a reference quadrilateral?

Reply:
I think in this case you should be able to simplify the code below and just use https://docs.makie.org/stable/examples/plotting_functions/surface/#surface with z constructed separately

Reply:
Actually i had it this way in the beginning. Then I decided to make it more general in order to avoid doubling of code. If you think it is better having two separate routines i can change it back. 
Where would you see the benefit in having 2 routines??

Reply:
Then, you don't need the `reference_coordinates` call and can just triangulate the boundaries. By that, you circumvent the problem of discretizing the reference domain of non-nodal interpolations

Reply:
I have changed it now and opposite to what I mentioned before, I think it even cleaned up the code a little bit.